src/net.cpp

```diff
@@ -2366,7 +2366,7 @@ static CNetCleanup instance_of_cnetcleanup;
 void CConnman::Interrupt()
 {
     {
-        std::lock_guard<std::mutex> lock(mutexMsgProc);
+        LOCK(mutexMsgProc);
```

Comment:
3af084a6ee2e52c660e92387bf97b024cf480c1a:
Why locking the mutex is needed here?